auth/auth.go

```diff
@@ -101,6 +107,33 @@ func (a ACL) VerifyPeerCertificateServer(rawCerts [][]byte, verifiedChains [][]*
 		return nil
 	}
 
+	input := map[string]interface{}{
+		"certificate": cert,
+	}
+
+	// Check against OPA
+	if a.AllowOPAQuery != nil {
+		results, err := a.AllowOPAQuery.Eval(context.TODO(), rego.EvalInput(input))
+		// rst, _ := json.MarshalIndent(results, "", "  ")
+		// fmt.Printf("%s\n", rst)
+		if err != nil {
```

Comment:
Let's use `rego.ResultSet.Allowed()` here, It will return true/false based on OPA's usual conventions:
https://pkg.go.dev/github.com/open-policy-agent/opa/rego#ResultSet.Allowed

I think this should work:
```
if err != nil {
  // error handling
}

if results.Allowed() {
  return nil
}
```